lib/content/utils.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const _ = require('lodash');
+const isUUID = require('validator/lib/isUUID');
+
+
+const check = {
+
+  /**
+   * Determine if an id is in uuid format; update session for error
+   *
+   * @param  {object} config global config
+   * @param {object} req - HTTP Request
+   *
+   * @returns {boolean} false-not in uuid format / true-it is!
+   */
+  id: (config, req) => {
+    if ((typeof req.params.id !== 'string') || (!isUUID(req.params.id))) {
+      _.set(req.session, '404', {
```

Comment:
This is the very definition of a side effect. We don't want magic side effects. Can we please have this just return true/false (or true/message) and have a separate helper that will set the 404 redirect? Especially as the 404 redirect is _identical_ except for the message?